duckduckgo/iOS
# remove center search and favorites option

Task/Issue URL: https://app.asana.com/0/392891325557410/1184981768664685
Tech Design URL:
CC:

**Description**:

Removes the center search and the option to disable favorites.

**Steps to test this PR**:
1. Migrate from an old version with center search enabled. 
1. Ensure browsing and new tab creation work as expected.



**Device Testing**:


**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
